tests/integration/deployment_test_cases.txt

```diff
@@ -0,0 +1,10 @@
+Testing 1 signed image deployment...;apply;tests/integration/deployments/deployment_i1.yaml;default;deployment.apps/i1-deployment created;
```

Comment:
would strongly recommend to use a better machine readable format (YAML, JSON) that will also be less fragile against strings with `;` and easy to edit via `yq` or `jq`. This is hard to read and worse to edit, I would think

Reply:
btw. would prefer yaml ;-)

Reply:
changed to yaml, have a look